ticker/public/index.html

```
@@ -196,6 +197,18 @@ <h3>Open games in other apps</h3>
               <p class="hint small" id="watchAppsStatus"></p>
             </section>
 
+            <section class="block drawer-section" data-panel="channels">
+              <h3>IPTV playlist (M3U)</h3>
+              <p class="hint">Import the M3U link from your IPTV provider (Xtream “m3u_plus” links work too). Game details then list your channels that carry the game, and open them in your player app — Core Line itself never plays video.</p>
+              <div class="feed-add">
+                <input class="focusable" id="playlistUrl" type="url" placeholder="https://provider.example/get.php?…m3u_plus" autocomplete="off" spellcheck="false">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)